pkg/handlers/scripts/install.sh

```diff
@@ -10,6 +10,44 @@ function fail {
 	echo "Error: $msg" 1>&2
 	exit 1
 }
+function prompts_can_prompt() {
+	# Need the TTY to accept input and stdout to display
+	# Prompts when running the script through the terminal but not as a subshell
+	if [ -t 1 ] && [ -c /dev/tty ]; then
+		return 0
+	fi
+	return 1
+}
+function read_installation_path {
+	if ! prompts_can_prompt ; then
+		echo "Automatically setting default path: ${OUT_DIR}, shell is not interactive"
+		return
+	fi
+	printf "Select installation path:[press 'Enter' to use default path ${OUT_DIR}]: \n"
+	filepath=
+	while true ; do
+		read -r -p "Path:  " filepath </dev/tty
+		filepath="${filepath:-$OUT_DIR}"
+		if [ -d "$filepath" ] ; then
+			break
+		fi
+		printf "$filepath is not a directory...\n"
```

Comment:
If the path they entered isn't a directory, I think we should exit with an error rather than using the default path. Because they didn't intend for the default path to be used.

Reply:
If the path is invalid, the script will prompt to enter valid path again until user enters a valid path/enters blank [default]
eg: 
```
➜  replicatedhq curl http://localhost:3000/kots | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  5109    0  5109    0     0    962      0 --:--:--  0:00:05 --:--:--  1248
Installing replicatedhq/kots v1.86.2 (https://github.com/replicatedhq/kots/releases/download/v1.86.2/kots_darwin_all.tar.gz)...
######################################################################## 100.0%
Enter installation path (leave blank for /usr/local/bin): aws
aws is not a directory...
Enter a valid installation path (leave blank for /usr/local/bin): /usr/bavan/bin
/usr/bavan/bin is not a directory...
Enter a valid installation path (leave blank for /usr/local/bin):
You don't have write permissions to /usr/local/bin
Do you want to enter your password to grant write permissions to /usr/local/bin [Y/n]? n
Please run the following command to complete the installation:
    sudo mv /tmp/tmpinstalldir/kots /usr/local/bin/kubectl-kots
```

Reply:
Should we exit after 3 attempts of invalid installation path? [similar to `sudo`, if password is wrong 3 times, sudo will exit with an err]

Reply:
exit on invalid path with 3 attempts
```
➜  replicatedhq curl http://localhost:3000/kots | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  5185    0  5185    0     0   380k      0 --:--:-- --:--:-- --:--:-- 1012k
Installing replicatedhq/kots v1.86.2 (https://github.com/replicatedhq/kots/releases/download/v1.86.2/kots_darwin_all.tar.gz)...
######################################################################## 100.0%
Enter installation path (leave blank for /usr/local/bin): aws
aws is not a directory...
Enter installation path (leave blank for /usr/local/bin): /usr/bavan/bin
/usr/bavan/bin is not a directory...
Enter installation path (leave blank for /usr/local/bin): /tmp/bin
/tmp/bin is not a directory...
============
Error: 3 invalid attempts. Exiting...
```

Reply:
Oh I see what you mean. Yeah, that sounds good to me.